client.cpp

```diff
@@ -423,7 +423,12 @@ void client::handle_event(short evtype)
         new_evtype = EV_READ;
     }
     if (evbuffer_get_length(m_write_buf) > 0) {
-        assert(finished() == false);
+        if (finished()) {
+            if (m_config->requests > 0 )
```

Comment:
@tomerr-lb curios: is it possible to run memtier and limiting both test run time and num of requests?

Reply:
It shouldn't be possible as there is no logic doing it ,but i'll test just in case